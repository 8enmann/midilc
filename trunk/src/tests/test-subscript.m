/** Tests subscripting.
  *
  * @author Ben
  */

main() {
  Note a;
  Sequence b;
  Chord c;
  Chord temp;
  Number s;
  a = A5w;
  b = new_sequence();
  a = (a as Chord) + C5 + Eb5;
  c = a;
  c[1] = A6;
  c[2] = A4;
  b = b + a;
  b = b + c;
  b = b + c;
  /* set the start time of a to the start time of the 3rd note */
  temp = b[2];
  s = temp.start;
  a.start = s;
  b[2] = a;
  play(b);
}
