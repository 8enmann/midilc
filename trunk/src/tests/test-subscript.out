0,16,81
0,16,72
0,16,75
16,16,81
16,16,93
16,16,69
32,16,81
32,16,72
32,16,75

